modules_basic/avatar/image.js

```diff
@@ -74,6 +76,24 @@ exports.create = function (api) {
       )
     },
 
+    avatar_image_src: function (author) {
+      var src = ready && avatars[author] ? api.blob_url(avatars[author].image) : genSrc(author)
+
```

Comment:
I'm not really sure if this pattern works (stollen from function below `avatar_image`). I.e. can you return a straight var early then modify it later?